kurtosis-module/impl/participant_network/mev_boost/mev_boost_launcher.go

```diff
@@ -1,18 +1,61 @@
 package mev_boost
 
-import "github.com/kurtosis-tech/kurtosis-core-api-lib/api/golang/lib/enclaves"
+import (
+	"fmt"
+	"strings"
+
+	"github.com/kurtosis-tech/kurtosis-core-api-lib/api/golang/lib/enclaves"
+	"github.com/kurtosis-tech/kurtosis-core-api-lib/api/golang/lib/services"
+	"github.com/kurtosis-tech/stacktrace"
+)
 
 type MEVBoostLauncher struct {
-	// TODO Analogous to GethLauncher, BesuLauncher, NimbusLauncher, etc.
+	RelayCheck bool
+	Relays     []string
 }
 
-/*
-KEVIN NOTE: You don't need this to be an interface yet because you only have one mev-boost node (the flashbots one),
-but if you got multiple in the future you'd probably want a MEVBoostLauncher interface with a Launch function, and
-then have implementations of it
- */
-func (launcher *MEVBoostLauncher) Launch(enclaveCtx enclaves.EnclaveContext) (*MEVBoostContext, error) {
-	// TODO A function for launching a MEV Boost node in the same way that GethLauncher
-	//  In the future, this will probably take in relay information as well
-	panic("TODO IMPLEMENT ME")
+const (
+	flashbotsMevBoostImage        = "flashbots/mev-boost"
+	flashbotsMevBoostPort  uint16 = 18550
+)
+
+var (
+	usedPorts = map[string]*services.PortSpec{
+		"api": services.NewPortSpec(flashbotsMevBoostPort, services.PortProtocol_TCP),
+	}
+	networkIdToName = map[string]string{
+		"5":        "goerli",
+		"11155111": "sepolia",
+		"3":        "ropsten",
+	}
+)
+
+func (launcher *MEVBoostLauncher) Launch(enclaveCtx *enclaves.EnclaveContext, serviceId services.ServiceID, networkId string) (*MEVBoostContext, error) {
+	containerConfigSupplier := func(string) (*services.ContainerConfig, error) {
+		relayCheckOption := ""
+		if launcher.RelayCheck {
+			relayCheckOption = "-relay-check"
+		}
+		networkName, ok := networkIdToName[networkId]
+		if !ok {
+			networkName = fmt.Sprintf("network-%s", networkId)
+		}
+		command := []string{
+			"mev-boost",
+			fmt.Sprintf("-%s", networkName),
+			relayCheckOption,
```

Comment:
I'm not sure but I thiiiiiink this might cause an error if should-check-relay = false, because then you'll be passing in a positional argument `""` to the `mev-boost` binary. I think (?) you'd instead want to do an if-check and add it to `command` only if should-check-relay is true